report.php

```diff
@@ -21,6 +21,9 @@
 require_once($CFG->dirroot . '/admin/tool/log/store/xapi/lib.php');
 require_once($CFG->dirroot . '/admin/tool/log/store/xapi/classes/form/reportfilter_form.php');
 
+$pagename = 'logstorexapierrorlog';
+admin_externalpage_setup($pagename);
```

Comment:
I don't think we have to add this because this set the page setup to admin page which is not good for us.

only what we need:
`
    require_login(null, false);
    if (!has_capability('moodle/site:config', context_system::instance())) {
        // The requested section could depend on a different capability
        // but most likely the user has inadequate capabilities
        print_error('accessdenied', 'admin');
    }
`

Reply:
If you set this a little bit down, around mform call you can use $systemcontext variable.